spec/rubocop/changes/checker_spec.rb

```diff
@@ -17,6 +17,11 @@
     ).run
   end
 
+  before do
+    allow(Rubocop::Changes::Shell).to receive(:run).with(
+      "git rev-parse --show-toplevel")
```

Comment:
Maybe we can add a spec covering this case.

Reply:
added